tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -301,3 +305,32 @@ def build_ports_dict(nsg, direction_key, ip_protocol):
                     ports[p] = IsAllowed
 
         return ports
+
+
+class AppInsightsHelper(object):
```

Comment:
FWIW, I am starting to think it is unnecessary for us to create these wrapper classes for static methods. We get the same effect with less overhead when we just declare and import the module functions where we need them.

@kapilt  maybe you have some thoughts on this?

Reply:
I think those wrapper classes serve us as a namespace replacement.
We can surely keep it as standalone function, but in that case I'd prefer us to keep it in different python modules.
Just makes code a little more self-descriptive

Reply:
Oops, I thought this class in its own file. Gotcha.

Reply:
Resolve after offline conversation (this is shared utils file)